var/spack/repos/builtin/packages/rocm-validation-suite/package.py

```diff
@@ -50,7 +51,9 @@ def setup_build_environment(self, build_env):
 
     def cmake_args(self):
         spec = self.spec
+        incl = spec['pciutils'].prefix.include
         args = ['-DHIP_INC_DIR={0}'.format(spec['hip'].prefix),
                 '-DROCM_SMI_DIR={0}'.format(spec['rocm-smi-lib'].prefix),
-                '-DROCBLAS_DIR={0}'.format(spec['rocblas'].prefix)]
+                '-DROCBLAS_DIR={0}'.format(spec['rocblas'].prefix),
+                '-DCMAKE_CXX_FLAGS=-I {0}'.format(incl)]
```

Comment:
Can you make CMake find this dependency instead of setting CMAKE_CXX_FLAGS?

Reply:
you mean something like find_package patch update on the CMakeLists.txt file?

Reply:
@haampie ping

Reply:
Hey @arjun-raj-kuppala, I quickly checked the `pciutils` spack package, and it does not install any header files at all, so this can't be the right fix. What problem are you trying to solve?